### PR TITLE
Create password validate account

### PR DESCRIPTION
**Pull request by GuenoleCHERUBIN**

### CODE REVIEW

**Review by EmileCalixte — COMMENTED** [review]
What is the views/site/password-created.php file ? Why is it empty ?

File: controllers/SiteController.php [hunk 1]
```diff
@@ -108,4 +112,59 @@ public function actionAbout()
...
+        Yii::$app->user->login($user);
+        $_SESSION["userPasswordHash"] = $user->password_hash;
+        return $this->redirect("/");
+    }
+
+    /**
+     * @param $token
+     * @return User|null
+     */
+    private function checkIfUserExist($token)
```
Comment (alexisljn) [inline controllers/SiteController.php]: You should name this method checkIfUserMatchToToken cause it's clearly what this method does.

Comment (EmileCalixte) [reply controllers/SiteController.php]: I fully agree

**Review by alexisljn — CHANGES_REQUESTED** [review]
Look at Comment of SiteController

File: controllers/SiteController.php (hunk 1 above)
Comment (GuenoleCHERUBIN) [reply controllers/SiteController.php]: True

**Review by EmileCalixte — APPROVED** [review]

**Review by alexisljn — APPROVED** [review]

**Pull request merged**